cookbook/integrations/aws/sagemaker_pytorch/requirements.txt

```diff
@@ -178,11 +178,13 @@ stringcase==1.2.0
     # via dataclasses-json
 tensorboardx==2.1
     # via -r requirements.in
-torch==1.8.0
+    # torch==1.8.1+cpu doesn't work currently.Hence using the wheel link directly.
```

Comment:
we want the gpu variant for the sample.

Reply:
The GPU variant requires  different base image as this change to use GPU version fails with current buster image.

```
ERROR: torch-1.8.1+cu111-cp39-cp39-linux_x86_64.whl is not a supported wheel on this platform.
WARNING: You are using pip version 20.2.3; however, version 21.1.1 is available.
```
It seems it  would require cuda installation which has the driver or toolkit to talk to the GPU and i couldn't find another smaller image than the one we currently  have `pytorch/pytorch:1.7.0-cuda11.0-cudnn8-devel`

I tried creating a CUDA image on top of buster by installing cuda toolkit but the installation went on forever and i aborted it. 
Please let me know if you have any suggestions here.

We can support the CPU image for CI since it breaks due to the size and document users to use CUDA image. I know its not proper solution but could be helpful for short term for breakages.